deployments/arbitrum-mainnet/.chainId

```diff
@@ -1 +0,0 @@
-42161
```

Comment:
do we want to keep this?

Reply:
No this folder has a typo in it `arbitrum_mainnet` not `arbitrum-mainnet`